client/components/Players/ShakaPlayer.vue

```diff
@@ -101,7 +102,9 @@ export default Vue.extend({
       shaka.polyfill.installAll();
 
       if (shaka.Player.isBrowserSupported()) {
-        this.player = new shaka.Player(this.$refs.shakaPlayer);
+        // We use a global for ease of debugging and to fetch data from the playback information popup
+        window.player = new shaka.Player(this.$refs.shakaPlayer);
```

Comment:
We probably want to have this settled up as undefined in beforeDestroy, like we do with muxjs